### DISCUSSION TITLE
Glad to see you found my research paper :)

### DISCUSSION THREAD

**Discussion by sam-scintam** [nicolai-wachenschwan/arc-overhang-prusaslicer-integration#20] [General]
Hi Nicolai,

Just seen CNC Kitchen's video - glad that you found my research paper and that it gave you some inspiration for solving the warping issues :) I'm not working directly in this area anymore, but I'd be happy to have a chat!

Great work on the arc overhangs, I look forward to using it in Prusaslicer!!

Sam

**Comment by nicolai-wachenschwan** [comment]
cool to hear you found my script :)
